api_app/analyzers_manager/observable_analyzers/checkdmarc.py

```diff
@@ -14,7 +14,7 @@ class CheckDMARC(classes.ObservableAnalyzer):
 
     def run(self):
         if not which(self.check_command):
-            self.report.status = self.report.Statuses.FAILED.name
+            self.report.status = self.report.Status.FAILED
             self.report.save()
```

Comment:
Lines 17 and 18 can be removed from here actually since we are raising `AnalyzerRunException`.